docs/README.md

```diff
@@ -41,6 +41,12 @@ Phone authentication has to be enabled in the Firebase console.
 You can find phone numbers to use for testing in the Firebase Console by navigating to
 Authentication > Sign-in method > Sign in providers > Phone > Phone numbers for testing
 
+## Generation OpenSource Licenses
+
+To generate the OpenSource licenses dart file via [flutter_oss_licenses](https://pub.dev/packages/flutter_oss_licenses), run the command `flutter pub run flutter_oss_licenses:generate.dart lib/presentation/licenses/oss_licenses.dart` from the project root.
+
+>Note: You need to format the [oss_licenses.dart](../lib/presentation/licenses/oss_licenses.dart) for `flutter analyze` to work.
```

Comment:
Change to "Run `flutter format .` after generating the OpenSource licences dart file"

Reply:
Ok